### DISCUSSION TITLE
vercel env pull - "The decrypt query parameter is deprecated", is this an issue?

### DISCUSSION THREAD

**Discussion by marcuscaum** [vercel/community#589] [Help] [answered]
`vercel env pull`  not working properly

This results in a useless `.env.local` file with all the strings wrong.

**Comment by sjsadowski** [comment]
100% this is an issue that started this afternoon. The .env files now contain the encrypted values, not the decrypted values. 'vercel dev' is also only supplying the encrypted values to the environment.

**Reply by sjsadowski** [reply]
The response I got from support:

>This is a known issue for CLI version 24.2.0. Our engineers have released a patch for our APIs and this issue should be fixed by the time you read this reply. You can also try to downgrade the CLI back to version 24.0.1.

And it is indeed working again.

**Comment by leerob** [comment, marked as answer]
Hey there. Apologies for this. A regression was introduced into CLI version `24.2.0`. We released a patch version and it is now fixed 🙌
